docs/markdown/Fs-module.md

```diff
@@ -34,6 +34,29 @@ by the string is a symbolic link.
 
 ## File Parameters
 
+### absolute
+
+Returns an absolute path based on the Meson source directory, relative to the directory where the
+current `meson.build` file is.
+If specified, a leading `~` is expanded to the user home directory.
```

Comment:
The unit test checks that this is the same as `meson.source_root() / 'foo'`. Whenever possible there should be only one way of doing something.

And the `~` expansion does not make sense here. The argument needs to be a relative path. Tildes are absolute.

Reply:
I meant for absolute to handle 3 common cases, where the goal is to avoid end users having to have extra if-elif:
* pass through already absolute paths
* expand ~ if leading the path
* Resolve relative paths to Meson.source_root